### ISSUE TITLE
"File Not Found" displays for "achecker.js" noted in cypress-accessibility-checker README.md

### ISSUE THREAD

**Issue by jmosley5** [IBMa/equal-access#2135]
**Describe the bug**
As a new user reading the `cypress-accessibility-checker README.md`, all links should redirect to the correct page where linked. In the ["Usage"](https://github.com/IBMa/equal-access/blob/master/accessibility-checker/src/README.md#usage) section, clicking `located here` in the following statement directs the user to a 404 not found page. 

```
Examples of how to use each of the APIs below can be found in the achecker.js test file [located here](https://github.com/IBMa/equal-access/blob/master/cypress-accessibility-checker/test/cypress/integration/achecker.js).
```

**To Reproduce**
1. Go to the[ "Usage"](https://github.com/IBMa/equal-access/tree/master/cypress-accessibility-checker#usage) section for the `cypress-accessibility-checker`
2. Click on "located here" in the sentence "Examples of how to use each of the APIs below can be found in the achecker.js test file [located here]"
3. The user is redirected to a 404 not found page
4. [Link](https://github.com/IBMa/equal-access/blob/master/cypress-accessibility-checker/test/cypress/integration/achecker.js)

**Expected behavior**
The user should be directed to the correct usage page.

**Screenshots**
<img width="1278" alt="Screenshot 2024-12-23 at 3 31 21 PM" src="https://github.com/user-attachments/assets/6cad9d33-5c56-4ee4-ad58-1c32c9b05d5f" />

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: Any
 - Version Any

**Smartphone (please complete the following information):**
N/A

**Additional context**
None at this time.

## Definition of Done

- [ ] Design artifacts reviewed
- [ ] Work item implemented
- [ ] Automated test case created (when applicable)
- [ ] Unit test completed
- [ ] Run Checker (when applicable)
- [ ] Manual and Screen Reader Testing (when applicable)
- [ ] Video recording of user experience walkthrough (if impact to UX)
- [ ] PR created
